docs/content/deployments/admin/deploying-on-netlify.md

```diff
@@ -100,7 +100,15 @@ After you authorize Netlify to use GitHub, you’ll be asked to pick the reposit
 
 ![Choose Repository](https://res.cloudinary.com/dza7lstvk/image/upload/v1668001871/Medusa%20Docs/Netlify/D0r6Q1e_th5uei.png)
 
-In the form that shows, keep all fields the same and click on the “Show advanced” button before the “Deploy site” button.
+A section titled "Basic build settings" will appear. There should be three inputs that display, with some default settings set. Ensure the fields are set as follows (you may need to change the publish directory):
+
+```
+Base directory: (leave blank)
+Build command: yarn build
+Publish directory: public
+```
```

Comment:
```suggestion
In the "Basic build settings" section, make sure the fields have the following values:

- Base directory: leave empty
- Build command: yarn build
- Publish directory: public

```

Reply:
Thank you for taking the time to respond. I feel that "make sure the fields have the following values," then, indicating "leave empty" with no text decoration (parentheses or otherwise), indicates to a reader that the base directory variable should be set to `leave empty`, though of course most users will probably read the meaning of words and leave the field empty. Yet, I still strive for annoying levels of clarity in documentation, especially considering how many non primary-english speakers probably use this excellent software. What do you think?

Reply:
Makes sense! Then you can change it to `(leave empty)`